### ISSUE TITLE
Sysbox doesn't seem to work on GKE 1.26.8-gke.200

### ISSUE THREAD

**Issue by tyranitar** [nestybox/sysbox#738]
Hi, I have a GKE cluster with control plane version `1.26.8-gke.200`. I created a node pool with the same k8s version and deployed the Sysbox installer `DaemonSet`. That seemed to work, as the node received the label `sysbox-runtime: running`. However, I found multiple occurrences of the following error in the installer logs:

```
couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
```

Afterwards, I tried deploying a container on the node and running `docker run hello-world`, but I got the following error:

```
Unable to find image 'hello-world:latest' locally
latest: Pulling from library/hello-world
719385e32844: Pull complete
Digest: sha256:4f53e2564790c8e7856ec08e384732aa38dc43c52f02952483e3f003afbf23db
Status: Downloaded newer image for hello-world:latest
docker: Error response from daemon: cgroups: cgroup mountpoint does not exist: unknown.
ERRO[0002] error waiting for container: context canceled
```

Afterwards, I tried going through the same steps on a node pool with version `1.24.16-gke.500`. That worked fine. Dataplane V2 was disabled for both scenarios.

Lastly, I wanted to ask about the documentation [here](https://github.com/nestybox/sysbox/blob/fc5c3c4fceb9ba44418fe2cf126ff3f1ecd3d584/docs/user-guide/install-k8s-distros.md#google-kubernetes-engine-gke). In particular it says for GKE:

> Create a cluster with Kubernetes v1.21-v1.23 releases.

Do the docs need to be updated or is it true that for GKE, only up to v1.23 is supported?

**Comment by rodnymolina** [comment]
@tyranitar, thanks for reporting this one.

Let me try to reproduce with this GKE release and I'll get back to you.

Oh, and yes, that's a doc bug we need to fix, GKE is supported beyond 1.23. Thanks for letting us know!

**Comment by ctalledo** [comment]
Hi @rahul-mereddy , just tried installing Sysbox on GKE 1.26 (`1.26.15-gke.1191000`) and it worked fine.

Closing this. If you see otherwise please re-open. Thanks!
